### ISSUE TITLE
Could you please give an example of your training and testing dataset?

### ISSUE THREAD

**Issue by ghost** [imlab-uiip/keras-segnet#6]
Hi, 
Your code is of great value for people who are new to DL in Keras and Segnet, and I really appreciate your work. I found you use _function **prep_data(mode)**_ to make label for the training and testing dataset, but there's not any example illustrating what's the .CSV file should look like, could you just kindly give an example of your training dataset and the CSV file corresponding to it? 

Thank you,
Tony

**Comment by eclique** [comment]
Hello, @SUZhaoyu. Thank you for your interest.
I have added data with corresponding CSV files. Keep in mind that you would still need to adapt `prep_data` for your data set, since it's not very universal.

**Comment by ghost** [comment]
Got it, thanks a lot!
